cloudsmith_cli/cli/commands/policy/license.py

```diff
@@ -0,0 +1,324 @@
+    """
+    List license policies.
+
+    This requires appropriate permissions for the owner (a member of the
+    organisation and a valid API key).
+
+    - OWNER: Specify the OWNER namespace (i.e. org)
+
+      Example: 'your-org'
+
+    Full CLI example:
+
+      $ cloudsmith policy license list your-org
+    """
+    owner = owner[0]
+
+    # Use stderr for messages if the output is something else (e.g.  # JSON)
+    use_stderr = opts.output != "pretty"
+
+    click.echo("Getting license policies ... ", nl=False, err=use_stderr)
+
+    context_msg = "Failed to get license policies!"
+    with handle_api_exceptions(ctx, opts=opts, context_msg=context_msg):
+        with maybe_spinner(opts):
+            policies, page_info = api.list_license_policies(
+                owner=owner, page=page, page_size=page_size
+            )
+
+    click.secho("OK", fg="green", err=use_stderr)
+
+    if utils.maybe_print_as_json(opts, policies, page_info):
+        return
+
+    print_license_policies(policies)
+
+
+@licence.command(aliases=["new"])
+@decorators.common_cli_config_options
+@decorators.common_cli_output_options
+@decorators.common_api_auth_options
+@decorators.initialise_api
+@click.argument("owner", default=None, required=True)
+@click.argument("policy_config_file", type=click.File("rb"), required=True)
+@click.pass_context
+def create(ctx, opts, owner, policy_config_file):
+    """
+    Create a new license policy in a namespace.
+
+    - OWNER: Specify the OWNER namespace (i.e. user or org) where you want
+      to create a license policy.
+
+        Example: 'your-org'
+
+    - POLICY_CONFIG_FILE: Config file specifying the settings for the
+      license policy to be created.
+
+        \b
+        Example:
+        {
+          "name": "your-license-policy",
+          "description": "your license policy description",
+          "spdx_identifiers" : ["your_licenses"],
+          "allow_unknown_licenses": false,
+          "quarantine_on_violation": true
+        }
+
+    Full CLI example:
+
+      $ cloudsmith policy license create your-org policy-config-file.json
+    """
+    # Use stderr for messages if the output is something else (e.g. JSON)
+    use_stderr = opts.output != "pretty"
+    policy_config = json.load(policy_config_file)
+
+    policy_name = policy_config.get("name", None)
+    if policy_name is None:
+        raise click.BadParameter(
+            "Name is a required field for creating a license policy.", param="name"
+        )
+
+    spdx_identifiers = policy_config.get("spdx_identifiers", None)
+    if spdx_identifiers is None:
+        raise click.BadParameter(
+            "SPDX Identifiers is a required field for creating a license policy.",
+            param="spdx_identifiers",
+        )
+
+    if len(spdx_identifiers) > len(set(spdx_identifiers)):
+        raise click.BadParameter(
+            "SPDX Identifiers must be unique.", param="spdx_identifiers"
+        )
```

Comment:
This isn't the case anymore iirc. The server will silently de-dupe them.

Reply:
TIL, will remove it now!